src/CalculatorUnitTests/UnitConverterViewModelUnitTests.cpp

```diff
@@ -284,6 +284,8 @@ for (unsigned int k = 0; k < categoryList->Size; k++)
         wstring unit1Name = vm->Unit1->Name->Data();
         for (unsigned int j = 0; j < unitList->Size; j++)
         {
+            if (i == j)
```

Comment:
a comment might help with understanding why i == j is a skip condition in this loop